themes/default/content/_index.md

```diff
@@ -1,13 +1,12 @@
 ---
-# The below values need updating.
 title: Pulumi - Universal Infrastructure as Code
 meta_desc: Pulumi's open source infrastructure as code SDK enables you to create, deploy, and manage infrastructure on any cloud, using your favorite languages.
 
 hero:
   title: [ "Universal", "Infrastructure as Code" ]
   description: |
-    Program the cloud and bridge the Dev and Ops divide with cloud engineering.
-    <br />From open source to enterprise.
+    Define infrastructure on any cloud using familiar programming languages.
+    For developers, infrastructure teams, and everybody in between.
```

Comment:
without the <br/> i believe this will display all on one line, which will be difficult to read

Reply:
It actually looks great to my eyes:
![image](https://user-images.githubusercontent.com/3953235/182463217-9d50b611-771a-424a-9d85-2375b316d60a.png)

In fact, the `<br>` has the effect of forcing awkward line breaks on different mobile screens.  For example:

![image](https://user-images.githubusercontent.com/3953235/182463339-e4e71e8b-ff5b-40bd-bed1-faef77297e41.png)

The enclosing div has a max width set, so I think we ought to just let the CSS do its thing?

Reply:
ah right i guess the container has a max width that's wrapping stuff. the dangling `For` isn't great, but no need to block.